en/docs/install-and-setup/perfromance-tuning-and-test-results/tuning-performance.md

```diff
@@ -116,25 +118,26 @@ The following diagram shows the communication/network paths that occur when an A
     If the Key Manager caching is enabled, the calls between the API Gateway and Key Manager are cached. As a result, the Key Manager is not invoked for each API call.
 
 -   **Client call API Gateway + API Gateway call Backend**
-    For backend communication, the API Manager uses PassThrough transport. This is configured in the `<API-M_HOME>/repository/conf/deployment.toml` file. For more information, see [Configuring passthru properties](https://docs.wso2.com/display/EI650/Tuning+the+HTTP+Transport) in the EI documentation. To configure socket timeout value, add following to the `deployment.toml`
-            ```java
-             [passthru_http]
-             http.socket.timeout=180000
+
+    For backend communication, the API Manager uses PassThrough transport. This is configured in the `<API-M_HOME>/repository/conf/deployment.toml` file. For more information, see [Configuring passthru properties](https://docs.wso2.com/display/EI650/Tuning+the+HTTP+Transport) in the EI documentation. Add the following section to the `deployment.toml` file to configure the Socket timeout value.
+            ``` java
+            [passthru_http]
+            http.socket.timeout=180000
```

Comment:
any minimum value for this ?

Reply:
The default value is mentioned in the infobox that follows this section. @shaniR , are we still missing anything?

"The default value for http.socket.timeout differs between WSO2 products. In WSO2 API-M, the default value for http.socket.timeout is 180000ms."

Reply:
the minimum could be 0 but I think that is an obvious value because timeouts cannot be minus values so I don't think we need to specifically mention it.

Reply:
That is not what I meant. This value should be greater than the value specified in worker_thread_keepalive_sec right?

Reply:
During an offline discussion, Bhathiya mentioned that this comment was not applicable.